src/main/java/ladder/domain/user/User.java

```diff
@@ -0,0 +1,23 @@
+package ladder.domain.user;
+
+public class User {
+	private final String name;
+
+	public User(String name) {
+		this.name = name;
+	}
+
+	public static User createUser(String name) {
+		if(name == null || name.isEmpty()){
```

Comment:
if 컨벤션 한번 확인해주시면 감사하겠습니다 😄 
```suggestion
		if (name == null || name.isEmpty()) {
```